R/legend.grob.R

```diff
@@ -142,6 +142,18 @@ legend.grob <- function(
 				legendi[['width']]  <- if (is.null(legendi[['width']])) { 2 } else { legendi[['width']] };
 
 				colorRamp <- colorRampPalette(legendi[['colours']]);
+				
+				labels.at <- if (is.null(legendi[['labels']])) legendi[['at']] else {
+				    n.labels <- length(legendi[['labels']]);
+				    max.value <- if (!is.null(legendi[['continuous.amount']])) legendi[['continuous.amount']] else 100;
+			        boundaries <- seq(0, max.value, length.out = n.labels + 1);
+
+			        sapply(
```

Comment:
Indent off here at least on github maybe just spaces/tabs mixed?

Reply:
Seems like it! Should be fixed now.

Reply:
Whitespace is still not good. There are a bunch of tabs before sapply

<img width="501" alt="image" src="https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general/assets/1639487/a57a6c8c-f8b6-44a2-a111-efb264ca178e">

Reply:
![Screen Shot 2023-05-23 at 9 47 25 AM](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general/assets/56950335/324dfe3b-3467-452a-89bd-dc0301c0e12f)
Seems to be fixed now. (Desregard the refactor - indentation should be the same before vs. after.)